src/TiltEstimator.cpp

```diff
@@ -25,72 +25,45 @@ const double gamma_const = 15;
 const double sampling_time_const = 0.002;
 
 static const std::string alpha_char = so::tools::toString(alpha_const);
-static const std::string beta_char  = so::tools::toString(beta_const);
+static const std::string beta_char = so::tools::toString(beta_const);
 static const std::string gamma_char = so::tools::toString(gamma_const);
 
 // Module specification
 // <rtc-template block="module_spec">
-static const char* TiltEstimator_spec[] =
-{
-  "implementation_id", "TiltEstimator",
-  "type_name",         "TiltEstimator",
-  "description",       "Tilt Estimator component",
-  "version",           "1.0",
-  "vendor",            "AIST",
-  "category",          "example",
-  "activity_type",     "DataFlowComponent",
-  "max_instance",      "10",
-  "language",          "C++",
-  "lang_type",         "compile",
-  // Configuration variables
-  "conf.default.alpha", alpha_char.c_str(),
-  "conf.default.beta", beta_char.c_str(),
-  "conf.default.gamma", gamma_char.c_str(),
-  ""
-};
+static const char *TiltEstimator_spec[] = {
+    "implementation_id", "TiltEstimator", "type_name", "TiltEstimator",
+    "description", "Tilt Estimator component", "version", "1.0", "vendor",
+    "AIST", "category", "example", "activity_type", "DataFlowComponent",
+    "max_instance", "10", "language", "C++", "lang_type", "compile",
+    // Configuration variables
+    "conf.default.alpha", alpha_char.c_str(), "conf.default.beta",
+    beta_char.c_str(), "conf.default.gamma", gamma_char.c_str(), ""};
 // </rtc-template>
```

Comment:
Please revert this modification as the new layout is less readable (maybe include guards to avoid reformatting).